src/Microsoft.ApplicationInsights/Extensibility/Implementation/Endpoints/Constants.cs

```diff
@@ -0,0 +1,32 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation.Endpoints
+{
+    /// <summary>
+    /// Endpoint Constants.
+    /// </summary>
+    internal static class Constants
+    {
+        /// <summary>Default endpoint for Breeze (aka Ingestion).</summary>
+        internal const string BreezeEndpoint = "https://dc.services.visualstudio.com/";
+
+        /// <summary>Default endpoint for Live Metrics (aka QuickPulse).</summary>
+        internal const string LiveMetricsEndpoint = "https://rt.services.visualstudio.com/";
+
+        /// <summary>Default endpoint for Profiler.</summary>
+        internal const string ProfilerEndpoint = "https://agent.azureserviceprofiler.net/";
+
+        /// <summary>Default endpoint for Snapshot Debugger.</summary>
+        internal const string SnapshotEndpoint = "https://ppe.azureserviceprofiler.net/";
```

Comment:
Please use agent.azureserviceprofiler.net (yes, it's the same as ProfilerEndpoint)

Reply:
Sure. Can you also reach out to Tomasz to update his spec?
All the other SDKs are coding off of that document.